include/mbedtls/x509_crl.h

```diff
@@ -96,6 +97,18 @@ typedef struct mbedtls_x509_crl
 mbedtls_x509_crl;
 
 /**
+ * \brief          Compare two CRL entry objects
+ *
+ * \param arg1     Pointer to the first entry to compare
+ * \param arg2     Pointer to the second entry to compare
+ *
+ * \return         an integer less than, equal to, or greater than zero if the
+ *                 first entry is respectively smaller than, equal to or greater
+ *                 than the second entry
+ */
+int mbedtls_x509_crl_entry_cmp( const void *arg1, const void *arg2);
```

Comment:
For the space before the '(': unfortunately I see this is a recurring mistake (will comment once only). I am not used to this particular codestyle and my vim plugin is not configured to help me here..sorry about that - will correct them.